.github/workflows/maintainer-absence.yml

```diff
@@ -0,0 +1,44 @@
+name: maintainer-absence
+
+on:
+  workflow_dispatch:
+    inputs:
+      startDate:
+        description: 'First day of maintainer absence [mm-dd-yyyy]'
+        required: true
+      endDate:
+        description:  'Last day of maintainer absence [mm-dd-yyyy]'
+        required: true
```

Comment:
You might get some complaints from @mjcheetham and other Europeans about this date format.

I tend to think that `yyyy-mm-dd` is pretty clear from context, but I'm willing to be wrong here. But of course, what matters is the parsing into a datetime in Javascript, which might not work right with that format. This is a completely optional thing to consider.

Reply:
I guess you could also use something like "6 Oct 2022" and Javascript groks it.

The suggestion in the label is good, because it raises awareness that specifying the date in dd-mm-yy format will either fail or worse: succeed but use the wrong date.

Reply:
Let's leave as-is for now. If we get complaints, we can re-visit.